src/components/AssetHeader/AssetChart.tsx

```diff
@@ -65,9 +66,15 @@ export const AssetChart = ({ accountId, assetId, isLoaded }: AssetChartProps) =>
   const { price } = marketData || {}
   const assetPrice = toFiat(price) ?? 0
   const [view, setView] = useState(accountId ? View.Balance : View.Price)
-  const filter = useMemo(() => ({ assetId, accountId }), [assetId, accountId])
   const translate = useTranslate()
 
+  const accountSpecifiers = useAppSelector(state => selectAccountSpecifier(state, asset?.caip2))
+  const accountSpecifier = accountSpecifiers?.[0] // TODO: maybe remove me, or maybe not
```

Comment:
FYI if reviewing this, this is coming from the current implementation of how we select the asset's accountSpecifier. We are guaranteed that it will always be the right one, but it'd be nice to refactor this in the future so as not to have to access the first array item but instead directly get the right `accountSpecifier`.

Reply:
Here ya go, both renamed and adding a selector consuming the first and returning the item at position 0